install.sh

```diff
@@ -59,8 +59,13 @@ install_app    lamp.sh
 install_app    bin/ncp/CONFIG/nc-nextcloud.sh
 run_app_unsafe bin/ncp/CONFIG/nc-nextcloud.sh
 systemctl restart mysqld # TODO this shouldn't be necessary, but somehow it's needed in Debian 9.6. Fixme
-install_app    ncp.sh
+ncp_rc=0
```

Comment:
what is all this exit code 5 stuff for? if this fails we should bail out I think

Reply:
It was my solution for the failing ncp-update, before you told me about `/.ncp-image`.

Should I simply create that file while the installer runs and delete it afterwards (I'll remove the exit 5 stuff once that's working)?

Reply:
Ah sorry, I saw your commit only now. I'll check it first (now I also understand your message on Telegram 🙈)

Reply:
cool, I am testing some fixes in devel right now

Reply:
I removed the `exit 5` parts :)